semantic-conventions/src/tests/data/markdown/stability/labels_expected.md

```diff
@@ -7,4 +7,12 @@
 | [`test.deprecated_attr`](labels_expected.md) | boolean | **Deprecated: Removed.**<br> |  | Required |
 | [`test.exp_attr`](labels_expected.md) | boolean |  |  | Required |
 | [`test.stable_attr`](labels_expected.md) | boolean |  |  | Required |
+| [`test.stable_enum_attr`](labels_expected.md) | string |  | `one` | Recommended |
+
+`test.stable_enum_attr` MUST be one of the following:
+
+| Value  | Description |
+|---|---|
+| `one` | member one |
+| `two` | member two |
```

Comment:
I wonder if it will be clear that the ones without a badge are experimental. I guess that's not a question for enums, but for all attributes we have today.

Reply:
We can start rendering experimental badges in addition to stable ones and include them everywhere, not just in the registry. Created issue on it https://github.com/open-telemetry/semantic-conventions/issues/763.
there is not much to do in this PR though - this is controlled by command line options set in the semconv repo.

Reply:
I think I can still add badge rendering to this PR and also make stability for  enums members required, so unresolving it

Reply:
updated tests to render experimental badges here, also added deprecated members